astropy/coordinates/tests/test_angles.py

```diff
@@ -436,29 +432,27 @@ def test_radec():
     # ra = Longitude("26:34:15.345634")  # unambiguous b/c hours don't go past 24
     # assert_allclose(ra.degree, 26.570929342)
     with pytest.raises(u.UnitsError):
-        ra = Longitude("26:34:15.345634")
+        Longitude("26:34:15.345634")
 
     # ra = Longitude(68)
     with pytest.raises(u.UnitsError):
-        ra = Longitude(68)
+        Longitude(68)
 
     with pytest.raises(u.UnitsError):
-        ra = Longitude(12)
+        Longitude(12)
 
     with pytest.raises(ValueError):
-        ra = Longitude("garbage containing a d and no units")
+        Longitude("garbage containing a d and no units")
 
     ra = Longitude("12h43m23s")
     assert_allclose(ra.hour, 12.7230555556)
 
-    # TODO: again, fix based on >24 behavior
-    # ra = Longitude((56,14,52.52))
-    with pytest.raises(u.UnitsError):
-        ra = Longitude((56, 14, 52.52))
-    with pytest.raises(u.UnitsError):
-        ra = Longitude((12, 14, 52))  # ambiguous w/o units
-    with pytest.warns(AstropyDeprecationWarning, match="hms_to_hours"):
-        ra = Longitude((12, 14, 52), unit=u.hour)
+    with pytest.raises(NotImplementedError, match="no longer supported"):
```

Comment:
If going with my suggestion, these would be removed (and equivalents doing in `test_create_tuples`).

Reply:
Done. Thanks!